ODPi-Runtime.md

```diff
@@ -233,6 +233,19 @@ The following requirements provide guidance around best practices of integrating
 * **[HCFS_INTEGRATION1]** First piece of Matt's doc goes here
 * **[HCFS_INTEGRATION2]** Second piece of Matt's doc goes here
 
+-   Applications SHOULD define APIs and behaviors explicitly in the FileSystem class instead of the HDFS class. A file system will be HCFS compatible if all the APIs are implemented from the FileSystem class. Defining API specification in the HDFS class is considered optional usage in HCFS distributed file system.
```

Comment:
This bullet is unclear and partly incorrect.  Perhaps it would be better to say:
* Applications SHOULD access file systems using the FileContext class APIs (which invoke AbstractFileSystem class methods), and SHOULD NOT directly access APIs from the HDFS classes.  
* A file system will be HCFS compatible if all the APIs are implemented from the AbstractFileSystem and FileSystem classes. Implementing APIs from the HDFS classes is considered optional usage in HCFS file systems.